tracker/processor.go

```diff
@@ -0,0 +1,20 @@
+package tracker
+
+import (
+	"github.com/vsariola/sointu"
+)
+
+type Processor struct {
+	*Player
+	playerProcessContext PlayerProcessContext
+	uiProcessor          EventProcessor
+}
+
+func CreateProcessor(player *Player, context PlayerProcessContext, uiProcessor EventProcessor) *Processor {
```

Comment:
Anything that looks like a constructor (taking bunch of parameters and returning a pointer to an object allocated in the heap) is usually called NewXXXX; here I would call this method NewProcessor